extensions/blocks/podcast-player/index.js

```diff
@@ -33,25 +33,47 @@ export const settings = {
 		_x( 'embed', 'block search term', 'jetpack' ),
 	],
 	supports: {
-		// Support for block's alignment (left, center, right, wide, full). When true, it adds block controls to change block’s alignment.
-		align: false /* if set to true, the 'align' option below can be used*/,
-		// Pick which alignment options to display.
-		/*align: [ 'left', 'right', 'full' ],*/
-		// Support for wide alignment, that requires additional support in themes.
+		/*
+		 * Support for block's alignment (left, center, right, wide, full). When
+		 * true, it adds block controls to change block’s alignment.
+		 */
+		align: false, // [ 'left', 'right', 'full' ]
+		/*
+		 * Support for wide alignment, that requires additional support in themes.
+		 */
 		alignWide: true,
-		// When true, a new field in the block sidebar allows to define an id for the block and a button to copy the direct link.
+		/*
+		 * When true, a new field in the block sidebar allows to define an id for
+		 * the block and a button to copy the direct link.
+		 */
 		anchor: false,
-		// When true, a new field in the block sidebar allows to define a custom className for the block’s wrapper.
+		/*
+		 * When true, a new field in the block sidebar allows to define a custom
+		 * className for the block’s wrapper.
+		 */
 		customClassName: true,
-		// When false, Gutenberg won't add a class like .wp-block-your-block-name to the root element of your saved markup
+		/*
+		 * When false, Gutenberg won't add a class like .wp-block-your-block-name to
+		 * the root element of your saved markup.
+		 */
 		className: true,
-		// Setting this to false suppress the ability to edit a block’s markup individually. We often set this to false in Jetpack blocks.
+		/*
+		 * Setting this to false suppress the ability to edit a block’s markup
+		 * individually. We often set this to false in Jetpack blocks.
+		 */
 		html: false,
-		// Passing false hides this block in Gutenberg's visual inserter.
-		/*inserter: true,*/
-		// When false, user will only be able to insert the block once per post.
+		/*
+		 * Passing false hides this block in Gutenberg's visual inserter.
+		 */
+		// inserter: true,
```

Comment:
Can this be  removed?

Reply:
Not sure, @retrofox can you confirm that?

Reply:
yes, I think it's ok.  👍

Reply:
Removed in https://github.com/Automattic/jetpack/pull/15281/commits/648c24036e7ddd4d61b9a5c30d4733d9cf6b3416